pyproject.toml

```diff
@@ -175,6 +175,8 @@ rich = "~13.4.2"
 towncrier = "~23.6.0"
 # Watch files for celery and mkdocs reload
 watchdog = "~3.0.0"
+# Factory Boy dependency. Don't upgrade to v26.x until breaking change is fixed https://github.com/joke2k/faker/issues/2070
+faker = ">=0.7.0,<26.0.0"
```

Comment:
This seems like a really wide range to me. Should we set a higher lower-bound?

Also, can you move this up so the dependencies stay in alphabetical order?

Reply:
`>=0.7.0` was copied from factoryboy, which is our only dependency that needs faker